### PR TITLE
RFC log format

### PR DESCRIPTION
**Pull request by ZenoTan**
### What problem does this PR solve?

Change log format according to RFC. Solve #2449 

### What is changed and how it works?

Rewrite `log.Fatal` or `log.Error`. Append rfc format error information to log. Use pingcap/errors.
RFC example is **[error="[PD:format:ErrFormatParseHistoryIndex] parse history index error"]**.
RFC code must have **[Registry:Class:Error]**.
Log output is like:
```
[2020/07/30 13:18:25.243 +08:00] [FATAL] [main.go:95] ["join meet error"] [error="join self is forbidden"] [error="[PD:storage:ErrStorageEtcdLoad] load config from storage etcd error"] [stack="github.com/pingcap/log.Fatal\n\t/Users/zeyuantan/go/pkg/mod/github.com/pingcap/log@v0.0.0-20200511115504-543df19646ad/global.go:59\nmain.main\n\t/Users/zeyuantan/Temp/my/err/cmd/pd-server/main.go:95\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"]
```

### Check List

Tests

- No code

Code changes

- Has log format change

Side effects

Related changes

#2658

### CODE REVIEW

File: client/base_client.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ package pd
 
 import (
 	"context"
+	errs "github.com/pingcap/pd/v4/pkg/errors"
```
Comment (disksing) [inline client/base_client.go:18]: em, why not name the package `github.com/pingcap/pd/v4/pkg/errs` directly?

Comment (ZenoTan) [reply client/base_client.go:18]: Originally, I used "errors" in #2658 . Then when I changed log code, I found github.com/pkg/errors was also used. So... = =

Comment (HunDunDM) [reply client/base_client.go:18]: Need to remove `pkg/errors`?

Comment (ZenoTan) [reply client/base_client.go:18]: `a`

Comment (ZenoTan) [reply client/base_client.go]: Some places use `pkg/errors` like `error.WithStack`.

**Pull request closed without merging**